scripts/swebench_rna_one.py

```
@@ -0,0 +1,1462 @@
+    )
+    state["selected_condition"] = condition
+    if readiness.exit_code != 0:
+        raise HarnessError(
+            "RNA pre-warm did not produce a queryable extracted graph; see prewarm logs"
+        )
+    return state, commands
+
+
+def collect_patch(checkout: Path) -> str:
+    completed = subprocess.run(
+        ["git", "diff", "--binary", "--no-ext-diff", "HEAD"],
```

Ruff (0.15.21)

[error] Starting a process with a partial executable path

(S607)